logic/constants.py

```diff
@@ -28,6 +28,10 @@
         ("SS", "Sandship"),
         ("FS", "Fire Sanctuary"),
         ("SK", "Sky Keep"),
+        (
+            "Triforce",
+            "Sky Keep",
+        ),  # for the purposes of restricting triforces to sky keep
```

Comment:
```suggestion
        # for the purposes of restricting triforces to sky keep
        ("Triforce", "Sky Keep"),
```
If you put the comment above the line it's commenting on instead of after it, the formatter won't break lines stupidly like that